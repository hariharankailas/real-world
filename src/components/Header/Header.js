/***********************************************************************************************
 * File : Header
 * Desc : Header with Home Link
 * Author: Hariharan Kailas
 * Date: 10-Jul-2021
 ***********************************************************************************************/
import React from 'react';
import { Link } from 'react-router-dom';
import "./Header.scss";
const Header = () => {
    return (
        <div>
            <Link to="/" className="nav-link text-center">
                REAL W<i className="bi bi-vinyl-fill"></i>RLD
            </Link>
        </div>
    );
}

export default Header;