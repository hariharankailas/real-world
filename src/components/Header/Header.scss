a.nav-link {
  font-size: 42px;
  color: white;
  background-color: cornflowerblue;

  transition: 1s font-size;
  &:hover {
    color: white;
    background-color: darkorange !important;
    font-size: 48px;
  }
  &:active,
  &:focus {
    color: white;
  }
  i {
    color: white;
    font-size: 32px;
  }
}
