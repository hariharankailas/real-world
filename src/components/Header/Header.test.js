import React from 'react';
import { shallow } from 'enzyme';
import Header from './Header';
describe('LandingPage', () => {
  it('should render correctly', () => {
    const component = shallow(<Header/>);
    expect(component).toMatchSnapshot();
  });
});