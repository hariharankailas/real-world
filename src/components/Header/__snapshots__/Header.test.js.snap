// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LandingPage should render correctly 1`] = `
<div>
  <Link
    className="nav-link text-center"
    to="/"
  >
    REAL W
    <i
      className="bi bi-vinyl-fill"
    />
    RLD
  </Link>
</div>
`;
